docs/Rakefile

```diff
@@ -4,6 +4,7 @@ require('yaml')
 
 desc "generate js from jsx"
 task :js do
+  system "cp ../node_modules/babel/node_modules/babel-core/browser.min.js ./_js"
```

Comment:
2 things here. 1 important and the other nitpicky…

1. The important piece: please copy to `js`, not `_js`. We don't want to run the file through babel afterwards :). The less important piece, it might be good to make this a separate rake task but I don't care so much as long as the release task works. I think we'll want to end up checking in the final file, we do now for React and JSXTransformer. cc @spicyj 

I'm going to be nitpicky here… Can we name the file `babel-browser.min.js`